bridgebot/main.py

```diff
@@ -13,3 +14,7 @@
     declarer = get_input_enum(Players, "declarer")
 
     trick_winners = cardplay.play(deck.deal(), contract, declarer)
+
+    score = get_score_from_result(contract, Doubles.NONE, trick_winners, False)
```

Comment:
I'll debug this tomorrow. But for now I'm hardcoding the doubles and the vulnerability